symfony/src/Controller/PatientStatusController.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Organizations;
+use App\Entity\PatientStatus;
+use App\Entity\PatientStatusHistory;
+use App\Entity\PatientStatusImport;
+use App\Entity\PatientStatusTemp;
+use App\Service\PatientStatusImportService;
+use App\Service\LoggerService;
+use App\Form\PatientStatusImportFormType;
+use App\Form\PatientStatusImportConfirmFormType;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Form\FormError;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\Routing\Annotation\Route;
+use Pmi\Audit\Log;
+
+/**
+ * @Route("/s")
+ */
+class PatientStatusController extends AbstractController
+{
+    /**
+     * @Route("/patient/status/import", name="patientStatusImport", methods={"GET","POST"})
+     */
+    public function patientStatusImport(Request $request, SessionInterface $session, EntityManagerInterface $em, LoggerService $loggerService, PatientStatusImportService $patientStatusImportService)
+    {
+        $form = $this->createForm(PatientStatusImportFormType::class);
+        $form->handleRequest($request);
+        if ($form->isSubmitted()) {
+            $file = $form['patient_status_csv']->getData();
+            $fileName = $file->getClientOriginalName();
+            $patientStatuses = [];
+            $patientStatusImportService->extractCsvFileData($file, $form, $patientStatuses);
+            if ($form->isValid()) {
+                if (!empty($patientStatuses)) {
+                    $organization = $em->getRepository(Organizations::class)->findOneBy(['id' => $session->get('siteOrganizationId')]);
+                    $patientStatusImport = new PatientStatusImport();
+                    $patientStatusImport->setFileName($fileName);
+                    $patientStatusImport->setOrganization($organization);
+                    $patientStatusImport->setAwardee($session->get('siteAwardeeId'));
+                    $patientStatusImport->setUserId($this->getUser()->getId());
+                    $patientStatusImport->setSite($session->get('site')->id);
+                    $patientStatusImport->setCreatedTs(new \DateTime());
```

Comment:
Minor style suggestion (would apply to other places as well)

```suggestion
                    $patientStatusImport = new PatientStatusImport();
                    $patientStatusImport
                        ->setFileName($fileName)
                        ->setOrganization($organization)
                        ->setAwardee($session->get('siteAwardeeId'))
                        ->setUserId($this->getUser()->getId())
                        ->setSite($session->get('site')->id)
                        ->setCreatedTs(new \DateTime());
```